File: 3136_valid-word/valid-word.c
bool isValid(char* s) {
    bool v=0,c=0;
    if(strlen(s)<3){
        return 0;
    }
    for(int i=0;i<strlen(s);i++){
        if((s[i]>='0'&&s[i]<='9')||(s[i]>='a'&&s[i]<='z')||(s[i]>='A'&&s[i]<='Z')){
            if(s[i]=='a'||s[i]=='e'||s[i]=='i'||s[i]=='o'||s[i]=='u'||s[i]=='A'||s[i]=='E'||s[i]=='I'||s[i]=='O'||s[i]=='U'){
                v=1;
            }else if(s[i]>'9'){
                c=1;
            }
        }else{
            return 0;
        }
    }
    if(c&&v) return 1;
    else return 0;
}

File: 3136_valid-word/valid-word.java
class Solution {
    public boolean isValid(String word) {
        boolean v = false, c = false;
        if (word.length() < 3){
            return false;
        }
        for(int i = 0; i < word.length(); i++){
            if ((word.charAt(i) >= '0' && word.charAt(i) <= '9') || ( word.charAt(i) >= 'a' && word.charAt(i) <= 'z' ) || ( word.charAt(i) >= 'A' && word.charAt(i) <= 'Z')){
                if (word.charAt(i) == 'a' || word.charAt(i) == 'e' || word.charAt(i) == 'i' || word.charAt(i) == 'o' || word.charAt(i) == 'u' || word.charAt(i) == 'A' || word.charAt(i) == 'E' || word.charAt(i) == 'I' || word.charAt(i) == 'O' || word.charAt(i) == 'U'){
                    v = true;
                }else if ( word.charAt(i) > '9'){
                    c = true;
                }
            }else{
                return false;
            }
        }
        if (c && v) return true;
        else return false;
    }
}